guide.md

```diff
@@ -820,7 +820,7 @@ The IMEI and IMSI can be traced back to you in at least six ways:
 
 Here is also a good YouTube video on this topic: DEFCON Safe Mode - Cooper Quintin - Detecting Fake 4G Base Stations in Real-Time <https://www.youtube.com/watch?v=siCk4pGGcqA> <sup>[[Invidious]](https://yewtu.be/watch?v=siCk4pGGcqA)</sup>
 
-**For these reasons, it is crucial to get dedicated an anonymous phone number and/or an anonymous burner phone with an anonymous pre-paid sim card that is not tied to you in any way (past or present) for conducting sensitive activities (See more practical guidance in [Getting an anonymous Phone number][Getting an anonymous Phone number:] section).**
+ **For these reasons, it is crucial to get dedicated an anonymous phone number and/or an anonymous burner phone with an anonymous pre-paid sim card that is not tied to you in any way (past or present) for conducting sensitive activities. It is also possible get an anonymous pre-paid but preferably dedicated number from specific free and paid online services accepting anonymous cryptocurrencies like Monero (Get more practical guidance here: [Getting an anonymous Phone number][Getting an anonymous Phone number:]).**

```

Comment:
Please spell check before you push commits.

Reply:
> Thank you. But this was not a spell checking PR. Wanted to limit the scope of it. Sorry if I missed this as part of the sync.

Only if the spelling errors are introduced by your changes, which they are were in this case. It's not often, but do try and fix those before they get to master.